.github/workflows/ci.yaml

```
@@ -22,7 +22,7 @@ jobs:
       - name: Set up JDK ${{ matrix.java-version }}
         uses: actions/setup-java@v2
         with:
-          distribution: adopt
+          distribution: zulu
```

AdoptOpenJDK named Eclipse Temurin as their successor [here](https://hub.docker.com/_/adoptopenjdk?tab=description) - we should use temurin images instead of zulu ones.

```suggestion
          distribution: temurin
```